src/test/java/com/puppycrawl/tools/checkstyle/api/LocalizedMessageTest.java

```diff
@@ -53,6 +56,14 @@
 
     private static final Locale DEFAULT_LOCALE = Locale.getDefault();
 
+    @Test
+    public final void testLocaleIsValid() {
+        assertThat("The requested language is not supported",
```

Comment:
Can we also check that if default is not "us", and language in in set of known translations, then message from ROOT and default should be different.

Reply:
issue is approved.
@pbludov , please do this change if that is simple, to prevent such issue in future.
If this is not simple, ok to postpone to separate issue (but please create it).

Reply:
> Can we also check that if default is not "us", and language in in set of known translations, then message from ROOT and default should be different.

done